packages/commons/src/fhir/transformers/fhir-builders.test.ts

```diff
@@ -289,677 +276,6 @@ test('all values in the death registration are found in the result bundle', () =
   }
 })
 
-test('should build a minimal FHIR registration document without error', async () => {
```

Comment:
This test was very much out of date. Minimal registration document is built without errors in the other tests as well